pgxn/neon/extension_server.c

```diff
@@ -45,7 +45,7 @@ neon_download_extension_file_http(const char *filename, bool is_library)
 		handle = alloc_curl_handle();
 
 		curl_easy_setopt(handle, CURLOPT_CUSTOMREQUEST, "POST");
-		curl_easy_setopt(handle, CURLOPT_TIMEOUT, 3L /* seconds */ );
+		curl_easy_setopt(handle, CURLOPT_TIMEOUT, 60L /* seconds */ );
```

Comment:
I wonder, should we set a separate `CURLOPT_CONNECTTIMEOUT` for a shorter period of time? From my personal experience, when connection hangs it usually hangs forever. So the combination of
- CURLOPT_CONNECTTIMEOUT of 3s
- CURLOPT_TIMEOUT of 60s
- and a few retries

works best in the real life [unstable] network environment.

Yet, on second thought, here we connect to the local HTTP server, so it probably doesn't matter. And this logic should rather land inside `compute_ctl`

Reply:
Let's add any improvements to this path in https://github.com/neondatabase/neon/pull/10801